AppCenterAnalytics/AppCenterAnalytics/TransmissionTarget/MSPropertyConfigurator.h

```diff
@@ -37,6 +37,54 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)setEventPropertyString:(NSString *)propertyValue forKey:(NSString *)propertyKey;
 
+/**
+ * Set a double event property to be attached to events tracked by this transmission target and its child transmission targets.
+ *
+ * @param propertyValue Property value. Must be finite (`NAN` and `INFINITY` not allowed).
+ * @param propertyKey Property key.
+ *
+ * @discussion A property set in a child transmission target overrides a property with the same key inherited from its parents. Also, the
+ * properties passed to the `trackEvent:withProperties:` override any property with the same key from the transmission target itself or its
```

Comment:
To be technically correct, this should be "`trackEvent:withProperties:` *or* `trackEvent:withTypedProperties:` overrides", yes?

Reply:
Yes right, updating it to add `trackEvent:withTypedProperties:`